com.unity.mobile.notifications/Editor/NotificationSettingsManagerEditor.cs

```diff
@@ -227,80 +215,82 @@ public override void OnGUI(string searchContext)
             if (width < k_SlotSize * 10)
                 width = k_SlotSize * 10;
 
-            var rect = new Rect(10f, 0f, width, Screen.height);
+            var totalRect = new Rect(k_Padding, 0f, width - k_Padding, Screen.height);
```

Comment:
a lot of magic numbers, was GUILayout not good?

Reply:
yes... 
I'll try GUILayout to remove the magic numbers in next pr...